mne/io/bti/bti.py

```diff
@@ -1060,42 +1016,57 @@ def __init__(self, pdf_fname, config_fname='config',
                  translation=(0.0, 0.02, 0.11), convert=True,
                  rename_channels=True, sort_by_ch_name=True,
                  ecg_ch='E31', eog_ch=('E63', 'E64'),
-                 verbose=None):
-
-        if pdf_fname is None:
-            raise ValueError('pdf_fname must be a path, not None')
+                 preload=None, verbose=None):
+        if preload is None:
+            warnings.warn('preload is True by default but will be changed to '
+                          'False in v0.12. Please explicitly set preload.',
+                          DeprecationWarning)
```

Comment:
One goes through the `warnings` module, the other through `logging`. We usually do `warnings.warn` for deprecations. I think it should appear for you if you have `preload=None` (default) when reading since we have this line in `utils.py`:
```
# force show of DeprecationWarning even on python 2.7
warnings.simplefilter('default')
```
Then I just noticed this in `decoding/base.py`:
```
            # We need deprecation warnings to always be on in order to
            # catch deprecated param values.
            # This is set in utils/__init__.py but it gets overwritten
            # when running under python3 somehow.
            warnings.simplefilter("always", DeprecationWarning)
```
Can you try replacing the line in `utils.py` with the one from decoding to see if it makes the warning appear for you? You'll probably need to restart your interpreter and re-import mne-python.